src/components/Charts/Graph.vue

```diff
@@ -169,12 +167,37 @@ export default defineComponent({
 
 		return { chartData, config, lineChart, graph, first_select, second_select, type }
 	},
+  watch:{
```

Comment:
In the repo root run this command and resolve any errors / warnings:
npm run lint